installinator/src/write.rs

```diff
@@ -665,6 +665,50 @@ impl ControlPlaneZoneWriteContext<'_> {
                 .register();
         }
 
+        engine
+            .new_step(
+                WriteComponent::ControlPlane,
+                ControlPlaneZonesStepId::CreateMeasurementDir,
+                "Creating measurement directory".to_string(),
+                async move |_cx| {
+                    std::fs::create_dir(
+                        self.output_directory.join("measurements"),
+                    )
+                    .map_err(|error| WriteError::CreateDirError { error })?;
```

Comment:
Do we want to ignore an error if this path already exists? Should we have a delete step, since this is a "mupdate"?

Reply:
Good suggestion on ignoring the error. The delete step should be covered by code further up the file where we clean everything out on install.